search/lib/stac/extensions/context.js

```diff
@@ -0,0 +1,12 @@
+function apply (result, { query, searchResult }) {
```

Comment:
so each extension now has to have a public method apply that takes the result that it extends, and the options that are passed in to `applyStacExtensions`